cmd/root.go

```diff
@@ -112,7 +112,11 @@ func rootRunE(cmd *cobra.Command, args []string) error {
 	}
 
 	if violations == 0 {
-		fmt.Fprintln(output.Stdout, "No violations found. Stay woke \u270a")
+		successExitMessage := "No violations found. Stay woke \u270a"
```

Comment:
Looks like `config` package loads directly from `yaml.Unmarshal` in `loadConfig` ignoring any viper configuration. where `rootRunE` calls `cfg, err := config.NewConfig(viper.ConfigFileUsed())` which is only a path reference.

Reply:
Ah yes, that's right. viper doesn't fully manage the config file yet, I had to restrict it to yaml because I haven't had the chance to port the unmarshaling over yet.